draft-kazuho-httpbis-priority.md

```diff
@@ -345,6 +336,31 @@ order the requests were generated.  Doing so maximizes the chance of the client
 making progress in using the composition of the HTTP responses at the earliest
 moment.
 
+
+# The Priority HTTP Header Field {#header-field}
+
+The Priority HTTP header field can appear in requests and responses. A client
+uses it to specify the priority of the response. A server uses it to inform
+the client that the priority was overwritten. An intermediary can use the
+Priority information from client requests and server responses to correct or
+amend the precedence to suit it (see {{merging}}).
+
+
+## urgency
```

Comment:
I do not think we need a sub-section just for each example.

Instead, assuming that we would have examples above, just having one another example in this section as a whole would be fine.